k8s-deploy.sh

```diff
@@ -95,6 +99,12 @@ then
   exit 1;
 fi
 
+# Check for invalid project
```

Comment:
Just a comment. The other upcoming projects, e.g. metis-statistics-dashboard will also require some script for their deployment.
Something to think about on how perhaps this script could be reusable for you?

Reply:
The stats dashboard is in another repository and so has its own script.  The projects share a deployment script here because it's a monorepo.